src/network/netdev/bridge.c

```diff
@@ -102,7 +103,7 @@ static int netdev_bridge_post_create(NetDev *netdev, Link *link, sd_netlink_mess
                         return log_netdev_error_errno(netdev, r, "Could not append IFLA_BR_PRIORITY attribute: %m");
         }
 
-        if (b->default_pvid > 0) {
+        if (b->default_pvid != VLANID_INVALID) {
```

Comment:
None is equivalent to 0 and hence disabled. It's rather a matter of taste if networkd shall accept both for configuration to disable the pvid. If the vid 0 would be valid on a bridge the above shown 'bridge vlan add vid 0 dev dummy0' would be accepted as well. That's what I tried to express in the comment above, sorry if it was confusing. 

The kernel disables the pvid 0 here http://elixir.free-electrons.com/linux/latest/source/net/bridge/br_vlan.c#L823.